src/libcrun/handlers/spin.c

```diff
@@ -0,0 +1,118 @@
+/*
+ * crun - OCI runtime written in C
+ *
+ * Copyright (C) 2017, 2018, 2019, 2020, 2021 Giuseppe Scrivano <giuseppe@scrivano.org>
```

Comment:
I think you can add your name here instead of mine :-)

Reply:
Sure, but most of the code originates from files with your name in the header 😅